File: Python/Arrays/L_nthFib.py
def fib(n: int) -> int:
        fib = [1,1]
        if n == 0:
            return 0
        if n <= 2:
            return fib[1]
        for i in range(3,n+1):
            nextFib = fib[0] + fib[1]
            fib[0] = fib[1]
            fib[1] = nextFib
        return fib[1]

File: cpp/Array/L_nthFib.cpp
#include<bits/stdc++.h>
using namespace std;

int fib(int n) {
    vector<int> fib = {1,1};
    int nextFib;
    if (n==0) return 0;
    if (n<=2) return fib[1];
    for (int i=3;i<=n;i++) {
        nextFib = fib[0] + fib[1];
        fib[0] = fib[1];
        fib[1] = nextFib;
    }
    return fib[1];
}

int main() {

}